lib/src/soloud.dart

```diff
@@ -751,11 +756,21 @@ class SoLoud {
     return (error: PlayerErrors.noError, length: ret);
   }
 
-  /// Seek playing in seconds
-  ///
-  /// [time] the time to seek
+  /// Seek playing in [time] seconds
+  /// [time]
   /// [handle] the sound handle
   /// Returns [PlayerErrors.noError] if success
+  /// 
+  /// NOTE: when seeking an mp3 file loaded using `loadIntoMem`=false
+  /// the seek operation is not performed due to lags. This occurs because the 
```

Comment:
Would it make more sense to perform the seek, even with the (large) lag? Keep the warning in the API. We can also log a warning into console every time the developer does this. But if the only issue is really "just" the lag, it seems prohibitive to completely deny this to the developer.

For example, I might load a background music mp3, play it, then fade it out to make room for some other jingle, and then seek to a position in the background music mp3 before fading it in again.

Reply:
On the other hand, if it's more code, or more brittle, to allow mp3 seeking in disk mode, then by all means let's just disallow it.

Reply:
That was my dilemma. I agree to enable the seek and write warnings in the doc/comments and into console logs.